secrets/geoserver_privileged_user_passwd.txt

```diff
@@ -0,0 +1 @@
+gerlsSnFd6SmM
```

Comment:
I understand this secret has to be somewhere for https://github.com/georchestra/georchestra-gateway/blob/f598de75e512ef1b621a8ac13372b5730eeb6759/gateway/docker-compose.yml to be runnable, but should not be the targetted datadir a default datadir ? In this case, maybe should the "secrets" files live in the neighbourhood of the dockercompose ? In a docker directory ?